### PR TITLE
Add rotate secrets lambda

### PR DESCRIPTION
**Pull request by MancunianSam**
- Add rotate secrets lambda.
- Remove duplicate name
- Rename the correct thing
- Rename variable
- Get the lambda arn in a different way

### CODE REVIEW

File: lambda/templates/rotate_keycloak_secrets_policy.json.tpl [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogStream",
```
Comment (TomJKing) [inline lambda/templates/rotate_keycloak_secrets_policy.json.tpl:7]: Could you put these in alphabetical order?

**Review by TomJKing — COMMENTED** [review]
Looks good. One very minor comment.

File: lambda/templates/rotate_keycloak_secrets_policy.json.tpl (hunk 1 above)
Comment (MancunianSam) [reply lambda/templates/rotate_keycloak_secrets_policy.json.tpl:10]: I've put these in order and the resources as well.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**